modules/ROOT/pages/pce-configuration.adoc

```diff
@@ -0,0 +1,17 @@
+= Configure Private Cloud Edition in Studio
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Studio 7.4 and later support native integration with Anypoint Platform Private Cloud Edition (PCE) versions 1.6 and 2.0.
+
+To point your Studio installation to integrate with the Anypoint platform components in a PCE deployment option:
```

Comment:
```suggestion
Follow these steps to integrate your Studio installation with the Anypoint Platform components in a PCE deployment:
```
Please read carefully to ensure that I did not accidentally change your intended meaning.